src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newArray = [];
+  const newState = { ...state };
```

Comment:
Let's make those variable names more descriptive and clear
```suggestion
  const states = [];
  const clone = { ...state };
```